source/renderer/app/config/timingConfig.js

```diff
@@ -3,15 +3,15 @@ const { isTest } = global.environment;
 
 // All configuration values for timeouts / intervals should go here
 /* eslint-disable max-len */
-export const REPORT_ISSUE_TIME_TRIGGER = isTest ? 8 : 5 * 60; // 5 minutes / 10 seconds (isTest = true) | unit: seconds
+export const REPORT_ISSUE_TIME_TRIGGER = isTest ? 12 : 5 * 60; // 5 minutes / 12 seconds (isTest = true) | unit: seconds
```

Comment:
@nikolaglumac @daniloprates 
When `isTest=true`, the `REPORT_ISSUE_TIME_TRIGGER` and `MAX_ALLOWED_STALL_DURATION` are set to 12 seconds and 15 seconds respectively. This ensures the tests for the loading screen don't need to run for 5 minutes in order to trigger the report issue notification.

Reply:
@MarcusHurney can we reduce both timeouts to just a couple of seconds for test environment?

Reply:
@nikolaglumac
Yes, that is possible, but it will cause the trouble connecting notification to flash on other acceptance tests that show the loading screen for more than a few seconds. It didn’t seem to be an issue when I ran the full suite of acceptance tests but the values I have set here are exactly the length of time needed for it not to flash on other tests. I am fine with making them shorter though.

Reply:
@MarcusHurney I have another idea - let's keep the default value for all other tests and then just set some minimum value like 1 or 2 seconds just for those new tests which actually test those notifications?

Reply:
I’ve been thinking about how to distinguish the two tests that use these time constants from the standard `isTest` environment variable that we typically use to differentiate test mode from develop etc, but I am not sure what the best approach is for determining that distinction. Do you know if there is a way to set a flag on `process.env` before each those tests run that I could then add to the `environment` variable and check for in `timingConfig.js`? I am open to other suggestions as well, that’s just one idea.

Reply:
@MarcusHurney I was thinking about the approach where we would set the 1 sec values as a part of a test "before" step. That way we could alter these timers only for these new tests.
Not sure if we can do that... Can you try it out?

Reply:
I can try that.